HomeTask4/HomeTask4/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+
+namespace HomeTask4
+{
+    /// <summary>
+    /// We accept from the comand line the path to the file
+    /// </summary>
+    class EntryPoint
+    {
+        static void Main(string[] args)
+        {
+            try
+            {
```

Comment:
Bad value name, "str" can't be an object of ParserForXmlFile, better name xmlParser